build_tools/github_actions/lint_version.sh

```diff
@@ -0,0 +1,57 @@
+# Copyright 2022 The StableHLO Authors.
```

Comment:
```
# Copyright 2023 The StableHLO Authors.
```

Reply:
good catch

Reply:
Should add that copyright lint one of these days..